### PR TITLE
Meetup styling on meetup day

### LINKED ISSUE #232: Meetup styling on meetup day
Currently, on the day of a meetup, the meetup's card does not indicate that it is happening on the current day.

@Saamiyah suggested to add some styling to the meetup card to "make it pop". @MrSunshyne suggested to add a live recording icon in red ([example](https://static.vecteezy.com/system/resources/thumbnails/008/471/382/small_2x/social-live-stream-linear-icon-web-streaming-badge-online-broadcast-news-thin-line-contour-button-symbol-isolated-outline-illustration-png.png)).

### PR DESCRIPTION
**Pull request by n-d-r-d-g**
Fixes #232.

I've modified the home page and meetups page for the day of a meetup.

To reproduce the changes locally (simulate the next meetup, i.e. 28 Sep 2024), replace the following functions in `useMeetups.ts`:
```ts
const upcomingMeetups = computed(() => {
  return sortedMeetups.value.filter((item) => {
    return isDateInFuture(new Date(item.Date), new Date('2024-09-28'))
  })
})

const todaysMeetups = computed(() => {
  return sortedMeetups.value.filter((item) => {
    return isDateToday(new Date(item.Date), new Date('2024-09-28'))
  })
})

const pastMeetups = computed(() => {
  const pastMeetupsData = sortedMeetups.value.filter((item) => {
    return isDateInPast(new Date(item.Date), new Date('2024-09-28'))
  })

  return pastMeetupsData.slice(0, pastMeetupsLimit)
})
```

Please find the screenshots below:

## Home page

**No meetup day:**
home_page_regular_day

**Meetup day:**
home_page_meetup_day

## Meetups page

**No meetup day:**
meetups_page_regular_day

**Meetup day:**
meetups_page_meetup_day

## Release Notes

- **New Features**
	- Introduced functionality to display meetups occurring today, enhancing user visibility of current events.
	- Added a "TODAY" label for events happening on the current date.
	- Updated event cards to reflect whether an event is upcoming or past with improved styling.

- **Bug Fixes**
	- Improved logic for determining event dates, ensuring accurate display of event statuses.

- **Refactor**
	- Streamlined code for accessing event properties, enhancing maintainability and readability.
	- Enhanced date comparison utilities for improved accuracy and clarity.
	- Improved the structure of the `EventCarousel` component for better readability.

- **Documentation**
	- Updated documentation to reflect new properties and functionalities related to meetups.

### CODE REVIEW

File: packages/frontendmu-nuxt/composables/useMeetups.ts [hunk 1]
```diff
@@ -23,30 +23,34 @@ export default function useMeetups({
...
-      return !isUpcoming(item.Date)
+      return isDateInPast(new Date(item.Date))
     })
 
     return pastMeetupsData.slice(0, pastMeetupsLimit)
   })
 
+  const nextMeetup = computed(() => {
+    return upcomingMeetups.value[upcomingMeetups.value.length - 1]
+  })
```
[the remark this answers is not in the document]
Comment (n-d-r-d-g) [reply packages/frontendmu-nuxt/composables/useMeetups.ts:48]: `sortedMeetups`, which is used in `upcomingMeetups`, is already sorted in the expected order.

File: packages/frontendmu-nuxt/components/cards/EventTilt.vue [hunk 2]
```diff
@@ -26,21 +27,18 @@ const tiltOptions = {
...
         <span
-          v-if="props.event.Date" class="inline-flex rounded-lg p-3 ring-4 ring-white dark:ring-white/10" :class="isUpcoming(props.event.Date)
-            ? 'bg-green-50 text-green-600 font-bold dark:bg-verse-900'
-            : 'bg-gray-50 text-gray-700'
-          "
+          v-if="event.Date" class="inline-flex rounded-lg p-3 ring-4 ring-white dark:ring-white/10 text-green-600 bg-green-50 dark:bg-verse-900 font-bold"
         >
           <Icon name="carbon:calendar" class="mr-2 h-6 w-6" />
-          <span>{{ new Date(props.event.Date).toDateString() }}</span>
+          <span>{{ new Date(event.Date).toDateString() }}</span>
```
[the remark this answers is not in the document]
Comment (n-d-r-d-g) [reply packages/frontendmu-nuxt/components/cards/EventTilt.vue:41]: The `isToday` check is unnecessary on the home page as there's no styling difference between the next meetup and today's meetup as only one of these two is displayed at any given point in time.

File: packages/frontendmu-nuxt/components/cards/EventTilt.vue [hunk 3]
```diff
@@ -59,15 +57,15 @@
...
+          v-if="event.Venue"
           class="flex gap-1 md:gap-0 items-center justify-start text-xl font-medium text-gray-500 dark:text-gray-100"
         >
           <Icon
             name="carbon:location"
             class="ml-[-1px] mr-1.5 h-5 w-5 flex-shrink-0 truncate text-gray-500 dark:text-gray-100"
             aria-hidden="true"
           />
-          <div>{{ props.event.Venue }}</div>
+          <div>{{ event.Venue }}</div>
```
[the remark this answers is not in the document]
Comment (n-d-r-d-g) [reply packages/frontendmu-nuxt/components/cards/EventTilt.vue:68]: The `isToday` check is unnecessary on the home page as there's no styling difference between the next meetup and today's meetup as only one of these two is displayed at any given point in time.

**Review by MrSunshyne — APPROVED** [review]

**Comment by MrSunshyne** [comment]
awesome! thanks for building this

**Pull request merged**